cabal-install/Distribution/Client/Install.hs

```diff
@@ -1439,7 +1436,7 @@ installUnpackedPackage verbosity installLock numJobs
                                       NoMultiInstance
                                       packageDBs ipkg'
 
-            return (Right (BuildResult docsResult testsResult ipkgs'))
+            return (Right (BuildResult docsResult testsResult (find ((==uid).installedUnitId) ipkgs')))
```

Comment:
So in what sense are we not reverting the convenience lib support in the old install code? This (+ change in the type) looks like we are reverting it.